WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/Add or Edit File/ProductDownloadFileViewController.swift

```diff
@@ -53,6 +60,28 @@ final class ProductDownloadFileViewController: UIViewController {
 //
 extension ProductDownloadFileViewController {
 
+    @objc private func presentMoreActionSheetMenu(_ sender: UIBarButtonItem) {
+        let menuAlert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        menuAlert.view.tintColor = .text
+
+        let deleteTitle = NSLocalizedString("Delete",
```

Comment:
Should we move this to a localization enum?

Reply:
Yes, this was already done in this other PR https://github.com/woocommerce/woocommerce-ios/pull/3014 👍